xdsl/irdl.py

```diff
@@ -1137,19 +1137,26 @@ def irdl_op_definition(cls: type[_OpT]) -> type[_OpT]:
 
     def optional_attribute_field(attribute_name: str):
 
-        @property
-        def impl(self: _OpT):
+        def field_getter(self: _OpT):
```

Comment:
why not just get and set?

Reply:
I think it's more verbose than just `get` and `set`. And the method names are never shown to the user anyway I believe.

Reply:
I have a slight preference for making functions sound like actions, so more like verbs. But I don't mind here, I agree with anton that users won't see it anyway.

Reply:
ok, great